org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/WizardConfiguration.xtend

```diff
@@ -32,6 +33,7 @@ class WizardConfiguration {
 	boolean needsGradleWrapper = true
 	JavaVersion javaVersion = JavaVersion.JAVA8
 	LanguageServer languageServer = LanguageServer.NONE
+	JUnitVersion junitVersion = JUnitVersion.JUNIT_4;
```

Comment:
Can we check for Junit5 in the TP and use that one if available?

Reply:
xtext-core has no idea about the TP. Will use JUnitVersion.DEFAULT here.
The wizard knows about presence of JUnit 5.